common/app/views/fragments/items/elements/facia_cards/image.scala.html

```diff
@@ -26,6 +26,6 @@
     }
     <!--[if IE 9]></video><![endif]-->
     @maybeImageMedia.map(ImgSrc.getFallbackUrl(_)).orElse(maybeSrc).orElse(maybePath).map { src =>
-        <img class="@RenderClasses(classes: _*)" alt="" src="@src"/>
+        <img loading="@{ if (Switches.LazyLoadImages.isSwitchedOn) "lazy" else "auto"}" class="@RenderClasses(classes: _*)" alt="" src="@src"/>
```

Comment:
Note, auto doesn't mean Chrome won't lazy load anything, it is simply the default (equivalent to not having a loading attribute present). It is possible to set it to 'eager' to explicitly indicate no loading here, but I think it's better to trust the default. We should review this as they roll out the feature and we see actual behaviour/data though.